stdlib/array/flux_gen.go

```diff
@@ -1,359 +1,3 @@
 // DO NOT EDIT: This file is autogenerated via the builtin command.
 
 package array
```

Comment:
Can you add a comment here that basically states: This package is empty but it ensures that each Flux package has a corresponding Go package. This simplifies importing Go packages with or without builtin values ?